vm/inc/ubpf.h

```diff
@@ -119,4 +127,14 @@ uint64_t ubpf_run(struct ubpf_vm *vm, void *mem, size_t mem_len, bool jit);
  * message will be stored in 'errmsg' and should be freed by the caller.
  */
 int ubpf_translate(struct ubpf_vm *vm, uint8_t *buffer, size_t *size, char **errmsg);
+
+/*
+ * Functions related to the instruction counting feature:
+ *  ubpf_enable_inst_cnt(struct ubpf_vm *vm): Enables instruction counting.
+ *  ubpf_get_inst_cnt(): Gets the instruction count struct.
+ */
+
+void ubpf_enable_inst_cnt(struct ubpf_vm *vm);
```

Comment:
Better to wait until we have a need for someone to disable it (I can't imagine why a program would need this). Otherwise it just adds dead code we'll never have coverage for.